extensions/pom.xml

```diff
@@ -53,7 +53,7 @@
         <dependency>
             <groupId>org.mule</groupId>
             <artifactId>mule-core-tests</artifactId>
-            <version>${project.version}</version>
+            <version>${mule.version}</version>
```

Comment:
Why do you need to do this? mule-core-tests is just another module in our project, so it must be referenced using project.version.